external/multi-base-images/spark/buildLightbendSpark.sh

```diff
@@ -45,11 +47,18 @@ $DIR/spark/dev/make-distribution.sh --name cloudflow-2.12 --tgz -Pscala-2.12 -Ph
 
 # build the Spark image
 tar -zxvf spark-$SPARK_VERSION-bin-cloudflow-2.12.tgz
-cd $DIR/spark/spark-$SPARK_VERSION-bin-cloudflow-2.12
+# replace protobuf jar to support sparksql-scalapb and load Scala-PB
+rm spark-$SPARK_VERSION-bin-cloudflow-2.12/jars/protobuf-java-*.*
```

Comment:
This script does not work for me when I just run it, it stops with:
```
VERSION=
```
executing parts of it in my shell does work. I'm ok with that now, it might have always been that way, or it is something in my shell. This should all be added to our normal build anyway, so we can fix it then.